### ISSUE TITLE
Prevent Existing Collaborator from Being Added Second Time

### ISSUE THREAD

**Issue by b-f-chan** [icgc-argo/dac-ui#696]
Currently, when adding a new collaborator, the UI should NOT let you re-add an existing collaborator (should validate that the email is already in use).

The UI allows you to make the add request, but ONLY shows an error AFTER you make the request.  Expected behaviour is that the request should not be possible and validation should reject immediately and show the error inline with the email field.

Basically, the behaviour should exist like validation for when user tries to add applicant as collaborator.

![image.png](https://images.zenhubusercontent.com/5c8bff46913aca2929d7074d/f5d3ae5e-2300-424d-8136-9b94c9367453)
